packages/relay/src/lib/eth.ts

```diff
@@ -326,7 +327,8 @@ export class EthImpl implements Eth {
     const requestIdPrefix = formatRequestIdMessage(requestId);
     this.logger.trace(`${requestIdPrefix} estimateGas()`);
     if (!transaction || !transaction.data || transaction.data === '0x') {
-      return EthImpl.gasTxBaseCost;
+      const toAccount = await this.mirrorNodeClient.getAccount(transaction.to);
+      return toAccount ? EthImpl.gasTxBaseCost : EthImpl.gasTxHollowAccountCreation;
```

Comment:
nit: comment.
```suggestion
      // when account exists return default base gas, otherwise return the minimum amount of gas to create an account entity
      return toAccount ? EthImpl.gasTxBaseCost : EthImpl.gasTxHollowAccountCreation;
```